test/test_vmap.py

```diff
@@ -2403,6 +2436,17 @@ def test_no_random_op_support(self):
                                         'vmap: We do not yet support calling random operations'):
                 vmap(op)(*args)
 
+    def test_conv2d(self):
```

Comment:
Can we add some more tests to convince ourselves that this works? 
From code reading everything looks fine to me, but it would be nice to make sure that different kernel sizes, strides, padding, dilation, and groups all work out

Reply:
The torchvision test in this file (resnet18 I think?) should exercise a lot of the cases already